tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
```

Comment:
You need to add here a comment with the linter output see: https://github.com/kubevirt/containerized-data-importer/pull/2774/files#diff-42fb4e8a81bc9dcc577c82ea3b473f3df7b7084a7277cf6e71b3ce2d5ba2dd94R29-R31